src/TorchSharp/NN/Module.cs

```diff
@@ -1120,6 +1111,59 @@ protected void Dispose(bool disposing)
                     }
                 }
             }
+
+            public interface IModule<T1, TResult>
```

Comment:
Nitpick which I am unsure if it makes sense: it is `Func<T,TResult>`, not `Func<T1, TResult>`.